### PR TITLE
Add keystore decodeing

### PR DESCRIPTION
**Pull request by godrei**
Add support for decoding PKCS12 Android Keystores (`DecodeKeystore`).

DecodeKeystore expects one certificate and one private key inside the PKCS12 keystore file. Each item is expected to have the same alias as the first bag attribute.

Sentinel errors are returned when keystore password, alias, or key passwords are incorrect.

### CODE REVIEW

File: pkcs12_test.go [hunk 1]
```diff
@@ -12,10 +12,97 @@ import (
...
+			storepass: "wrongpassword",
+			alias:     "key0",
+			keypass:   "keypass",
+			error:     IncorrectKeystorePasswordError.Error(),
+		},
+		{
+			name:      "Wrong key password",
+			pth:       filepath.Join("testdata", "example.jks"),
+			storepass: "storepass",
+			alias:     "key0",
```
Comment (lpusok) [inline pkcs12_test.go:57]: Is it possible to have multiple keys in a keystore?

File: pkcs12_test.go [hunk 2]
```diff
@@ -12,10 +12,97 @@ import (
...
+			} else {
+				require.NoError(t, err)
+				require.NotNil(t, gotPrivateKey)
+				require.NotNil(t, gotCertificate)
+			}
+		})
+	}
+}
+
+func TestDecodeKeystore2(t *testing.T) {
```
Comment (lpusok) [inline pkcs12_test.go]: Is this test needed?

**Review by ofalvai — DISMISSED** [review]

File: pkcs12_test.go (hunk 1 above)
Comment (godrei) [reply pkcs12_test.go:57]: In theory, you could store multiple certificates and keys in a keystone, but a single certificate and a single key are used for Android code signing keystores.
When you create a new keystore in Android Studio, you don't have other options. 
Also, I remembered that earlier Android docs mentioned `keytool` for generating keystores (and I suspected that with keeytool you have more options), but keytool is not mentioned anymore for generating keystores: https://developer.android.com/studio/publish/app-signing.html#sign-apk.

File: pkcs12_test.go (hunk 2 above)
Comment (godrei) [reply pkcs12_test.go]: No, this is just a leftover, I removed it here: https://github.com/bitrise-io/go-pkcs12/pull/3/commits/2a804477368dda191fffc6166159e1cd20907f56

**Review by lpusok — DISMISSED** [review]

**Comment by lpusok** [comment]
Can you please give access to the CI?

**Comment by godrei** [comment]
@lpusok added the team to the CI

**Review by lpusok — APPROVED** [review]

**Pull request merged**